app/views/admin/chapters/index.html.haml

```diff
@@ -0,0 +1,45 @@
+%section#banner
+  .row
+    %h1 Chapters
+    = link_to t('.view_all_chapters'), admin_chapters_path
+    &nbsp;
+    = link_to t('.view_active_chapters'), admin_chapters_path(active: true)
+    &nbsp;
+    = link_to t('.view_inactive_chapters'), admin_chapters_path(active: false)
+    .row
+      %hr
+      .large-4.columns
+        Name / City / Slug
+      .large-4.columns
+        Email / Twitter
+      .large-1.columns
+        Active?
+
+  - @chapters.each do |chapter|
+    .row
+      %hr
+      .large-4.columns
+        = link_to admin_chapter_path(chapter) do
+          %p= chapter.name
+        %p= chapter.city
+        %p= chapter.slug
+      .large-4.columns
+        %p= chapter.email
+        %p= chapter.twitter
+      .large-1.columns
+        = check_box_tag :chapter_active, 'true', chapter.active?, class: 'chapter_active', data: { 'chapter-id': chapter.id, }
+
+- content_for :page_footer do
```

Comment:
I don't mind having javascript available as well, however it should be optional and not the default. Also, a confirmation button to verify disabling chapters before any actions take place.